src/content/docs/iast/coverage.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: New Relic IAST Coverage
+tags:
+    - IAST
+    - New Relic IAST
+    - Coverage
+metaDescription: Find out how xxxx.
+---
+
+import iastCoverageOverview from 'images/iast_screenshot-crop_coverage-overview.webp'
+
+import iastCoverageSummary from 'images/iast_screenshot-crop_coverage-summary.webp'
+
+
+Through the coverage page you can see how many of your applications have or haven't been tested for exploitable vulnerabilities, as well as an overview of the health of all your applications. Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** and click **Coverage** on the left navigation pane to open the page. 
+
+If you want to test an application under **Untested applications** for exploitable vulnerabilities, click **Set up IAST** to [enable the untested application](/docs/iast/applications/#enable-iast).
+
+<img
+    title="Coverage for all your applications."
+    alt="Coverage for all your applications."
+    src={iastCoverageOverview}
+/>
+
+<figcaption>
+    **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST > Coverage**.
+</figcaption>
+
+
+## Details and remediation of an application [#remediation-app]
+
+Check out all the issues detected with our New Relic IAS testing on your applications.
```

Comment:
I think we want to delete "New Relic" wherever we can. "our" ties IAS to New Relic pretty clearly